wp/lib/bap_wp/src/environment.ml

```diff
@@ -62,7 +63,7 @@ type t = {
   int_handler : int_spec;
   loop_handler : loop_handler;
   exp_conds : exp_cond list;
-  arch : Arch.t;
+  arch : Theory.target;
```

Comment:
Is it possible to change the name of this field to `target`?

Reply:
Tedious (it changes the API), but possible.

Unless you don't want to change the name of the `mk_env` argument, but then what's the point?

Reply:
I'll change it.